src/main/java/org/isf/patvac/gui/PatVacEdit.java

```diff
@@ -58,6 +56,11 @@
 import org.isf.vaccine.model.Vaccine;
 import org.isf.vactype.manager.VaccineTypeBrowserManager;
 import org.isf.vactype.model.VaccineType;
+import java.awt.BorderLayout;
```

Comment:
These imports are out of order; they should be nearer to line 26

Reply:
Thanks!